pkg/vz/vz_driver_darwin.go

```diff
@@ -0,0 +1,62 @@
+package vz
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/lima-vm/lima/pkg/store/filenames"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Code-Hex/vz/v3"
+	"github.com/lima-vm/lima/pkg/driver"
+	"github.com/lima-vm/lima/pkg/limayaml"
+)
+
+type LimaVzDriver struct {
+	*driver.BaseDriver
+
+	machine *vz.VirtualMachine
+}
+
+func New(driver *driver.BaseDriver) *LimaVzDriver {
+	return &LimaVzDriver{
+		BaseDriver: driver,
+	}
+}
+
+func (l *LimaVzDriver) Validate() error {
+	if *l.Yaml.MountType == limayaml.NINEP {
+		return fmt.Errorf("field `mountType` must be %q or %q for VZ driver , got %q", limayaml.REVSSHFS, limayaml.VIRTIOFS, *l.Yaml.MountType)
+	}
+	return nil
+}
+
+func (l *LimaVzDriver) CreateDisk() error {
+	if err := EnsureDisk(l.BaseDriver); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (l *LimaVzDriver) Start(ctx context.Context) error {
+	logrus.Infof("Starting VZ (hint: to watch the boot progress, see %q)", filepath.Join(l.Instance.Dir, filenames.SerialLog))
+	vm, err := startVM(ctx, l.BaseDriver)
+	if err != nil {
+		return err
+	}
```

Comment:
I think error handling would be required to check version compatibility. Given that lima-vm will be built with the latest OS, handling of `vz.ErrBuildTargetOSVersion` is unnecessary. (maybe)
https://github.com/Code-Hex/vz#version-compatibility-check

Reply:
@balajiv113 Right. However, will be returned error might be not helpful for user side. So I meant to return an error with message which is helpful that this driver cannot use below macOS 13.

Reply:
Done